docs/js/react-dom.js

```diff
@@ -5950,7 +5950,7 @@ ReactDOMComponent.Mixin = {
         } else if (props.is) {
```

Comment:
This is a built file for the current stable version, so it will get reverted the next time it's built.
Let's omit this file.